nilearn/plotting/html_surface.py

```diff
@@ -365,7 +366,7 @@ def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
         If it is a number only values of amplitude greater
         than threshold will be shown.
         If it is a string it must finish with a percent sign,
-        e.g. "25.3%", and only values of amplitude above the
+        e.g. "25.3%%", and only values of amplitude above the
```

Comment:
Curious: why do we have to double the % ?

Reply:
Conceptually it works it allows "escaping" to actually print a percent sign.

without it I was getting those kind of errors during

- [testing](https://github.com/nilearn/nilearn/actions/runs/6240697742/job/16941334286#step:8:4226) 

<details> <summary> See error </summary>

<pre>

nilearn/_utils/docs.py:1084: in fill_doc
    f.__doc__ = docstring % indented
E   TypeError: not enough arguments for format string

During handling of the above exception, another exception occurred:
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/runner.py:341: in from_call
    result: Optional[TResult] = func()
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/runner.py:372: in <lambda>
    call = CallInfo.from_call(lambda: list(collector.collect()), "collect")
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/doctest.py:567: in collect
    module = import_path(
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/pathlib.py:567: in import_path
    importlib.import_module(module_name)
/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:961: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:961: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
<frozen importlib._bootstrap>:1014: in _gcd_import
    ???
<frozen importlib._bootstrap>:991: in _find_and_load
    ???
<frozen importlib._bootstrap>:975: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:671: in _load_unlocked
    ???
<frozen importlib._bootstrap_external>:843: in exec_module
    ???
<frozen importlib._bootstrap>:219: in _call_with_frames_removed
    ???
nilearn/reporting/__init__.py:9: in <module>
    from nilearn.reporting.glm_reporter import make_glm_report
nilearn/reporting/glm_reporter.py:23: in <module>
    from nilearn.plotting import plot_glass_brain, plot_roi, plot_stat_map
nilearn/plotting/__init__.py:66: in <module>
    from .html_surface import view_img_on_surf, view_surf
nilearn/plotting/html_surface.py:327: in <module>
    def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
nilearn/_utils/docs.py:1088: in fill_doc
    raise RuntimeError(f"Error documenting {funcname}:\n{str(exp)}")
E   RuntimeError: Error documenting view_surf:
E   not enough arguments for format string

</pre>

</details>

- [doc build](https://github.com/nilearn/nilearn/actions/runs/6240697737/job/16941343769#step:34:123)

<details> <summary> See error </summary>

<pre>

Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/generate.py", line 455, in generate_autosummary_docs
    name, obj, parent, modname = import_by_name(entry.name)
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/__init__.py", line 650, in import_by_name
    raise ImportExceptionGroup('no module named %s' % ' or '.join(tried), exceptions)
sphinx.ext.autosummary.ImportExceptionGroup: no module named nilearn.plotting.find_cut_slices

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/events.py", line 96, in emit
    results.append(listener.handler(self.app, *args))
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/__init__.py", line 808, in process_generate_options
    generate_autosummary_docs(genfiles, suffix=suffix, base_path=app.srcdir,
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/generate.py", line 460, in generate_autosummary_docs
    name, obj, parent, modname = import_ivar_by_name(entry.name)
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/__init__.py", line 707, in import_ivar_by_name
    real_name, obj, parent, modname = import_by_name(name, prefixes)
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/ext/autosummary/__init__.py", line 650, in import_by_name
    raise ImportExceptionGroup('no module named %s' % ' or '.join(tried), exceptions)
sphinx.ext.autosummary.ImportExceptionGroup: no module named nilearn.plotting

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/cmd/build.py", line 285, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/application.py", line 268, in __init__
    self._init_builder()
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/application.py", line 339, in _init_builder
    self.events.emit('builder-inited')
  File "/usr/share/miniconda3/envs/testenv/lib/python3.9/site-packages/sphinx/events.py", line 107, in emit
    raise ExtensionError(__("Handler %r for event %r threw an exception") %
sphinx.errors.ExtensionError: Handler <function process_generate_options at 0x7f0f0ccc8e50> for event 'builder-inited' threw an exception (exception: no module named nilearn.plotting)

Extension error (sphinx.ext.autosummary):
Handler <function process_generate_options at 0x7f0f0ccc8e50> for event 'builder-inited' threw an exception (exception: no module named nilearn.plotting)
Failed to import nilearn.reporting.
Possible hints:
* RuntimeError: Error documenting view_surf:
not enough arguments for format string
* AttributeError: module 'nilearn' has no attribute 'reporting'
Failed to import nilearn.plotting.
Possible hints:
* AttributeError: module 'nilearn' has no attribute 'plotting'
* RuntimeError: Error documenting view_surf:
not enough arguments for format string
make: *** [Makefile:82: html-noplot] Error 2

</pre>

</details>

I tried to create some MWE to show the problem but this is taking longer than expected so long story short: doing this fixed the problem. #SometimesPragmatismWins